pkg/cmd/teamcity-trigger/main.go

```diff
@@ -50,40 +101,24 @@ func main() {
 		log.Printf("created teamcity build (buildID=%s, branch=%s, opts=%+v): %s",
 			buildID, branch, opts, build)
 	})
-}
-
-func getBaseImportPath() string {
-	if bazel.BuiltWithBazel() {
-		return "./"
+	if count == 0 {
+		log.Fatal("no builds were created")
 	}
-	return "github.com/cockroachdb/cockroach/pkg/"
 }
 
 func runTC(queueBuild func(string, map[string]string)) {
-	buildID := "Cockroach_Nightlies_Stress"
-	if bazel.BuiltWithBazel() {
-		buildID = "Cockroach_Nightlies_StressBazel"
-	}
-	baseImportPath := getBaseImportPath()
-	importPaths := gotool.ImportPaths([]string{baseImportPath + "..."})
-
+	importPaths := gotool.ImportPaths([]string{"./" + "..."})
```

Comment:
Nit: Doesn't need to be done in this PR, but it is probably advantageous to use `bazel query` to enumerate `go_test` targets here rather than this function.

Reply:
I am now also wondering if there's a reason to pass maxTime to stress? It doesn't do anything right? Bazel/Go timeouts are enough.

Reply:
Nevermind we need it to give `stress an extra minute to clean up.`

Reply:
Ready for another look